### PR TITLE
Custom card container

### PR DESCRIPTION
**Pull request by mathildegrimal**
#### Before submitting, make sure:
- [x] You have rebased your branch
- [x] You describe your changes
- [x] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

##### Describe below How to test:
git fetch
git checkout custom-card-container

### CODE REVIEW

File: src/components/common/SubMenu.tsx [hunk 1]
```diff
@@ -14,21 +15,15 @@ export default function SubMenu({ navigation }: SubMenuProps) {
     { text: 'A propos', path: 'About' },
     { text: 'Blog', path: 'Blog' },
     { text: 'Partenaires', path: 'Partners' },
+    { text: 'Challenges', path: 'Challenges' },
+    { text: 'Astuces', path: 'Tips' },
   ];
```
Comment (Antonin-dev) [inline src/components/common/SubMenu.tsx:20]: config

File: src/screens/Blog/Blog.tsx [hunk 2]
```diff
@@ -1,11 +1,47 @@
 import React from 'react';
 import { View, Text } from 'react-native';
 import GlobalStyles from '@utils/GlobalStyles';
-interface BlogProps {}
-export default function Blog({}: BlogProps) {
+import CardContainer from '@common/CardContainer';
+import CustomView from '@common/CustomView';
+export default function Blog() {
+  const headerImage = require('../../../assets/images/blog.png');
+  const cards = [
```
Comment (Antonin-dev) [inline src/screens/Blog/Blog.tsx:8]: config

File: src/screens/Challenges/Challenges.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import GlobalStyles from '@utils/GlobalStyles';
+import CardContainer from '@common/CardContainer';
+import CustomView from '@common/CustomView';
+
+export default function Challenges() {
+  const headerImage = require('../../../assets/images/challenges.png');
+  const cards = [
+    {
```
Comment (Antonin-dev) [inline src/screens/Challenges/Challenges.tsx:10]: config

File: src/screens/Tips/Tips.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import GlobalStyles from '@utils/GlobalStyles';
+import CardContainer from '@common/CardContainer';
+import CustomView from '@common/CustomView';
+
+export default function Tips() {
+  const headerImage = require('../../../assets/images/tips.png');
+  const cards = [
+    {
+      text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Culpa cumque debitis dolor neque possimus.',
```
Comment (Antonin-dev) [inline src/screens/Tips/Tips.tsx:10]: config

File: src/components/common/SubMenu.tsx (hunk 1 above)
Comment (mathildegrimal) [reply src/components/common/SubMenu.tsx:20]: done

**Review by Antonin-dev — APPROVED** [review]

**Pull request merged**